resident-default.properties

```diff
@@ -263,11 +305,11 @@ mosip.iam.module.login_flow.name=authorization_code
 ###########################################################
 # IdP config
 # Note: When enabling MOSIP-IdP comment Mock Keycloak config, vise versa.
-################## MOSIP IdP config
-mosip.iam.module.clientID=G27lCI-trl4_pwXWBTb4WyCi0cCUZdnA5yVh3EGMBCg
+################## MOSIP ESignet config
+mosip.iam.module.clientID=U0Ip-je0CJWOtaUo9JOlkf7XVq_KRvRPKeVGw8b6N2c
```

Comment:
Please keep the old value here: 
`mosip.iam.module.clientID=G27lCI-trl4_pwXWBTb4WyCi0cCUZdnA5yVh3EGMBCg`